android/src/com/mapswithme/maps/editor/EditorHostFragment.java

```diff
@@ -301,6 +311,13 @@ public void onClick(View v)
           saveMapObjectEdits();
         }
         break;
+      case PHONE:
+        final String phone = ((PhoneFragment) getChildFragmentManager().findFragmentByTag(PhoneFragment.class.getName())).getPhone();
+        if (Editor.nativeIsPhoneValid(phone)) {
```

Comment:
Opening brace should be at the beginning of the next line according to project code style. Please, check in other places.